tvtk/pyface/ui/qt4/scene.py

```diff
@@ -338,7 +339,7 @@ def __init__(self, parent=None, **traits):
         super(Scene, self).__init__(parent, **traits)
 
         # Setup the default picker.
-        self.picker = picker.Picker(self)
+        self.picker = picker.Picker(weakref.proxy(self))
```

Comment:
I would think that making using a weak ref inside an __init__ method is less explicit than using a WeakRef trait in Picker. It will also mean that we avoid cycles between the scene and the picker in general. For example through out the life of the Scene object the picker could be replaced 

```python
my_picker = Picker(scene)
scene.picker = my_picker
```